client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -465,15 +465,15 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
 			shouldStickyNavButtons={ isCopySiteFlow( flow ) }
 			hideBack={ ! isCopySiteFlow( flow ) }
 			hideSkip={ true }
-			flowName="linkInBio"
+			flowName={ isCopySiteFlow( flow ) ? ( flow as string ) : 'linkInBio' }
```

Comment:
We could create a flowProps to differentiate both flows:

```jsx
	const flowProps = isCopySiteFlow( flow )
		? {
				flowName: flow as string,
				hideBack: false,
				shouldStickyNavButtons: true,
				goBack: () => exitFlow?.( '/sites' ),
		  }
		: {
				flowName: 'linkInBio',
				hideBack: true,
				shouldStickyNavButtons: false,
		  };
```

and the pass it destructured:

``` jsx
isWideLayout={ true }
{ ...flowProps }
```